topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,455 @@
+
+> <tip-title>Overfitting</tip-title>
+>Overfitting occurs when the network fails to generalize well to unseen data because it begins to model the noise and randomness in the training dataset.
+>![alt](../../images/galaxy-ludwig/explain_overfitting.png "Overfitting curve")
+>
+{: .tip}
+
+> <tip-title>Underfitting</tip-title>
+>Underfitting happens when the patterns in the training dataset are too complex for our model. It can’t capture them, so it has a poor performance on training and validation sets. 
+>![alt](../../images/galaxy-ludwig/explain_underfitting.png "Underfitting curve")
+>
+{: .tip}
+
+![loss vs epochs](../../images/galaxy-ludwig/test1_combined_loss.png "Loss versus Epochs during training. The graph illustrates the relationship between the training loss and the number of epochs during the training of the model.")
+
+Based on the training curve (blue line), we can draw the following conclusions:
+
+- CONSISTENT IMPROVEMENT:
+    Training loss consistently decrease over the five epochs. This indicates that the model is effectively learning and improving its performance on the dataset.
+- APPROACHING CONVERGENCE:
+    By Epoch 5, the training loss has reduced to less than 0.5. The gradual reduction in losses suggests that the model is approaching convergence, where further training may yield diminishing returns in terms of performance improvement.
+
+## Accuracy
+
+Plotting accuracy over epochs in deep learning is a common practice to visualize how well a model is learning to make correct predictions as training progresses. The x-axis represents the number of epochs, which are complete passes through the training dataset, while the y-axis shows the accuracy, typically expressed as a percentage.
+As training progresses, the accuracy on the training data usually increases, indicating that the model is learning to correctly classify more examples. 
+
+> <tip-title>Plotting accuracy and/or Loss</tip-title>
+> Plotting accuracy and plotting loss are both essential for monitoring the performance of a deep learning model, but they serve different purposes and provide different insights. While accuracy provides a straightforward measure of model performance, loss offers deeper insights into the training process, making both plots valuable for different aspects of model evaluation.
+{: .tip}
+
+![alt](../../images/galaxy-ludwig/test1_label_accuracy.png "Accuracy versus Epochs. This graph depicts the accuracy of the model over the course of training. The training accuracy curve indicates how well the model is performing on the training data.")
+
+Based on the training curve (blue line), we can draw the following conclusions:
+
+- THE DATASET IS RELATIVELY EASY FOR THE MODEL TO LEARN:
+    The model starts with a relatively high training accuracy of >0.8.
```

Comment:
Accuracy on test data should also be mentioned or cross-validation accuracy to decide a model really learns well.

Reply:
I added an additional information point to highlight this result and its conclusion.